libMBIN/Source/NMS/Globals/GcAISpaceshipGlobals.cs

```diff
@@ -225,7 +226,9 @@ public class GcAISpaceshipGlobals : NMSTemplate
         /* 0x928 */ public float TradeRouteFlickerFreq;
         /* 0x92C */ public float TradeRouteFlickerAmp;
         /* 0x930 */ public int TradeRouteMaxNum;
-        /* 0x938 */ public List<Colour> TradeRouteColours;
+        [NMS(Size = 0x4, Ignore = true)]
+        /* 0x934 */ public byte[] Padding934;
+        /* 0x938 */ public List<Vector4f> TradeRouteColours;
```

Comment:
Are you sure this isn't a colour? The name does have colour in it after all...? Seems weird for it to be a Vector4f...

Reply:
Yes:
    .text:1407ecf8a 48 8d 8f        LEA        RCX, [RDI + 0x938]
    .text:1407ecf91 44 0f b6 ce     MOVZX      R9D, SIL
    .text:1407ecf95 4c 8d 05        LEA        R8, [s_TradeRouteColours_1423b0d70]              ; = "TradeRouteColours"
    .text:1407ecf9c 48 8b d3        MOV        RDX, RBX
    .text:1407ecf9f e8 8c fd        CALL       NMS_Vector4f_0x10_RGBA_list                      ; undefined NMS_Vector4f_0x10_RGBA_list()

Reply:
I mean, that's what you have called that particular function, but this List type is used only by things which are colours, so I think it would be correct to have this as `List<Colour>`, and then change `ShipHUDTargetArrowsColour` in the UI globals back to `List<Colour>`
Having this as a Vector4f I think is not correct and will be confusing

Reply:
it is not, thisa one has RGBA as fields